### PR TITLE
[SMALLFIX] Sync code change in master into 1.3

### PR DESCRIPTION
**Pull request by apc999**
@jsimsa @calvinjia 
after this PR, master and branch-1.3 are only different in version related strings.

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -6,9 +6,8 @@ kramdown:
 
 # These allow the documentation to be updated with new releases of Alluxio.
 ALLUXIO_RELEASED_VERSION: 1.2.0
-# This would be "master" for the master branch, 1.1.1 for the 1.1.1 release, and
-# 1.1.1-RC2 for the second 1.1.1 release candidate.
-ALLUXIO_MASTER_VERSION_SHORT: 1.3.0
+# For example, "master" for the master branch, and 1.1 for 1.1.x releases and release candidates.
+ALLUXIO_MASTER_VERSION_SHORT: master
```
Comment (ifcharming) [inline docs/_config.yml]: Is this intended? shall we keep it `1.3.0` ?

Comment (apc999) [inline docs/_config.yml]: Updated

**Comment by apc999** [comment]
@ifcharming could you take another look?

**Comment by jsimsa** [comment]
LGTM for everything except for Chinese docs

**Pull request merged**